SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -46,20 +46,20 @@ handle_netcdf_file() {
     echo "" | notify_by_email $BACKUP_RECIPIENT  "Processing new underway CO2 file '$file'"
 
     local tmp_file_with_sig
-    local checks='cf imos:1.3'
+    local checks='cf imos:1.4'
 
     if is_imos_soop_co2_file $file; then
         tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
     elif is_future_reef_map_file $file; then
         checks='cf'
         tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
     else
-        file_error_and_report_to_uploader $BACKUP_RECIPIENT "Not an underway CO2 file '$file'"
+        file_error_and_report_to_uploader $BACKUP_RECIPIENT "Not an underway CO2 file " `basename $file`
```

Comment:
Sorry... I think this way the file name will become a third argument to the function and not included in the message. You need to remove the space between them:
```
file_error_and_report_to_uploader $BACKUP_RECIPIENT "Not an underway CO2 file "`basename $file`
```
Or put the filename within the double quotes:
```
file_error_and_report_to_uploader $BACKUP_RECIPIENT "Not an underway CO2 file `basename $file`"
```